pkg/controllers/apiservice.go

```diff
@@ -64,52 +116,18 @@ func createAPIHeadlessSvc(cfg *config.MicroshiftConfig, svcName string, svcPort
 			},
 		},
 	}
-	_, err = client.Services("default").Get(context.TODO(), svc.Name, metav1.GetOptions{})
+
+	_, err := client.Services("default").Get(context.TODO(), svc.Name, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
-		klog.Infof("Creating service %s", svc.Name)
+		logrus.Infof("creating svc %s", svc.Name)
```

Comment:
why logrus? no idea if it was originally in my patch, could be: bad Ajo! X'D

Reply:
My bad too, the original PR was before the klog replacement.